server/src/services/appVersion.js

```diff
@@ -0,0 +1,69 @@
+const Schmervice = require('@hapipal/schmervice')
+const AppVersionModel = require('../models/v2/AppVersion')
+const { executeQuery } = require('../query/executeQuery')
+
+const getAppVersionQuery = knex =>
+    knex('app_version')
+        .innerJoin(
+            knex.ref('app_version_localised').as('avl'),
+            'avl.app_version_id',
+            'app_version.id'
+        )
+        .innerJoin(
+            knex.ref('app_channel').as('ac'),
+            'ac.app_version_id',
+            'app_version.id'
+        )
+        .innerJoin(knex.ref('channel'), 'ac.channel_id', 'channel.id')
+        .select(
+            'app_version.id',
+            'app_version.version',
+            knex.ref('app_version.app_id').as('appId'),
+            knex.ref('app_version.created_at').as('createdAt'),
+            knex.ref('app_version.source_url'),
+            knex.ref('app_version.demo_url'),
+            knex.ref('channel.name').as('channel'),
+            knex.ref('ac.min_dhis2_version'),
+            knex.ref('ac.max_dhis2_version')
```

Comment:
If we add some `.as()` calls here then we could remove the `.rename()` calls in the model/definition file

```suggestion
            knex.ref('app_version.source_url').as('sourceUrl'),
            knex.ref('app_version.demo_url').as('demoUrl'),
            knex.ref('channel.name').as('channel'),
            knex.ref('ac.min_dhis2_version').as('minDhis2Version'),
            knex.ref('ac.max_dhis2_version').as('maxDhis2Version')
```